doc/user/content/concepts/indexes.md

```diff
@@ -26,6 +26,9 @@ view](/concepts/views/#non-materialized-views) causes view results to be
 to queries within the cluster, even for queries that do not use the index
 key(s).
 
+Indexes preserve monotonicity information.  As such, for indexed
+non-materialized views, results can be monotonic.
```

Comment:
I would be tempted to say "As such, for indexed views results can be monotonic", striking the "non-materialized views". There are several reasons a thing can be non-monotonic, one of which is it being a materialized view, but it's only one among many (no aggregations, no temporal filters, no non-monotonic inputs).

I think this may derive from me being not wholly bought in to "non-materialized view" as a synonym for "view".

Reply:
👍  -- The whole "non-materialized" was meant to be a transitional placeholder  to break people away from defaulting to creating materialized views.  But per your suggestion, aligning on vocabs now seems good to do.  Will update.

Reply:
The changes are also inline with what Marta has previously mentioned in an earlier review.  So, it's been percolating in my mind, and heh ... I guess now is the time 😄